#!/usr/bin/python3
def raise_exception(message=""):
    raise NameError(message)
